ivy/container/random.py

```diff
@@ -58,3 +58,57 @@ def random_uniform(
             map_sequences,
             out=out,
         )
+
+
+    # randint
+    @staticmethod
+    def static_randint(
+        low: Union[int, ivy.Container] = 0.0,
+        high: Union[int, ivy.Container] = 1.0,
+        shape: Optional[Union[int, Tuple[int, ...], ivy.Container]] = None,
+        device: Optional[Union[ivy.Device, ivy.NativeDevice, ivy.Container]] = None,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.randint. This method simply wraps the
+        function, and so the docstring for ivy.randint also applies to this method
+        with minimal changes.
+
+        Examples
+        --------
+        With no device argument and no out argument specified and int arguemnt for shape:
+
+        >>> x = ivy.Container.randint(low=ivy.Container(a=1), high=ivy.Container(b=10), shape=2)
+        >>> print(x)
+        ivy.array([5,7])
+
+        With no device argument and no out argument specified and sequence for shape:
```

Comment:
Please also update these headers with headers like `With one ivy.Container input`, `With multiple ivy.Container inputs` similar to the ones mentioned in the guide [here](https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html)